src/main/java/step3/RacingCarApplication.java

```diff
@@ -0,0 +1,28 @@
+package step3;
+
+import java.util.ArrayList;
+
+public class RacingCarApplication {
+
+    public static void main(String[] args) {
+        InputView inputView = new InputView();
+        InputDto input = inputView.getInput();
+        ResultView resultView = new ResultView();
+
+        ArrayList<RacingCar> carList = new ArrayList<>();
```

Comment:
자바코드를 보다보면 타입은 보통 인터페이스를 선언하는 경우가 많아요. 🙂 타입이 유연하면 이후에 구현체를 바꾸기에 편하기떄문입니다. 예를들어 List의 구현체를 이후에 LinkedList로 바뀌더라도 타입은 같으니 타입의 수정없이 변경할 수 있습니다!
```suggestion
        List<RacingCar> carList = new ArrayList<>();
```